examples/main_profile_feature.go

```diff
@@ -0,0 +1,79 @@
+// to run the CPU profiling: go build -ldflags "-X main.RunCPUProfile=true" main.go && ./main
+// to run the Mem profiling: go build -ldflags "-X main.RunMemProfile=true" main.go && ./main
+
+package main
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+
+	"github.com/optimizely/go-sdk/optimizely/client"
+	"github.com/optimizely/go-sdk/optimizely/decision"
+	"github.com/optimizely/go-sdk/optimizely/entities"
+	"github.com/optimizely/go-sdk/optimizely/notification"
+
+	"github.com/pkg/profile"
+)
+
+// stressTest has everything that test app has. it is used to run profile
+func stressTest() {
+	/*
+		For the test app, the biggest json file is used with 100 entities.
```

Comment:
Similar comment about the test app not relevant to the OS community.

Reply:
ok, addressed

Reply:
still referencing "test app"